stmhal/boards/PYBV10/stm32f4xx_hal_conf.h

```diff
@@ -47,6 +47,7 @@
 /* Exported constants --------------------------------------------------------*/
 
 #define USE_USB_FS
+//#define USE_USB_HS

```

Comment:
I was attempting to make the code as much of a "refactor" as possible and by only defining USE_USB_FS it keeps it functionally equivalent.
For pybv10/stm32f4xx_hal_conf.h the define is there to show its a compile option for when further testing is needed.
For other <boards>/stm32f4xx_hal_conf.h they'll need to be defined as need when modified.